16/nonickid/top_apple_songs.py

```diff
@@ -0,0 +1,57 @@
+# Simple program to received Apple Music Top Songs based on country's code provided
+# To received Top Songs list Apple's API is used
+# To received Country list country.io API is used. The data is used to map country code to country name
+import requests
```

Comment:
I usually split the modules as per pep8: first stdlib modules, blank line, external modules